src/tools/textEncryptor/textEncryptor.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const fs = require('fs');
+
+class TextEncryptor {
+  constructor(seed) {
+    if (typeof(seed) !== 'number')
+      throw new Error(
+        `Seed is expected to be a number. Its type is ${typeof seed}`
+      );
+    Object.defineProperties(this, {
+      _seed: {
+        value: seed,
+      },
+      _enternalSeed: {
+        value: 99241492,
+      },
+    });
+    Object.freeze(this);
+  }
+
+  _modifyCode(code) {
+    return code ^ (this._seed ^ 1 ^ this._enternalSeed);
+  }
+
+  encryptString(data) {
+    // eslint-disable-next-line eqeqeq
+    if (data == false) {
+      throw new Error('There is no data to encrypt!');
+    }
+    const newData = [];
+    for (let i = 0; i < data.length; i++) {
```

Comment:
Why not turn data into an array and iterate using .map?
Though your implementation is good too
```suggestion
    const newData = data.split('').map((char) => {
      const code = this._modifyCode(char.charCodeAt(0));
      return String.fromCharCode(code);
    };
```